.github/workflows/release.yml

```diff
@@ -23,10 +23,7 @@ jobs:
 
             - name: Build JS
               run: npm run build
-
-            - name: Zip files
-              run: zip release.zip ./* -r
-
+              
             - name: Tar files
               run: tar --exclude='release.zip' -czf release.tar.gz *
```

Comment:
Not important but you could also remove the exclude, since it's shouldn't be needed anymore

Reply:
Fixed and squashed with the relevant commit :)